content-repo/extra-docs/packs/Malware_Investigation_&_Response.md

```diff
@@ -0,0 +1,163 @@
+- The primary **incident type**: **Malware Investigation and Response**
+
+#### Fetch Data from an EDR integration
+1. In [MarketPlace](https://xsoar.pan.dev/marketplace), select a fetching integration (for more details on supported EDR integrations, check the [Endpoint Integrations](#endpoint-integrations) section):  
+   1. PaloAlto Cortex XDR 
+   2. Microsoft Defender For Endpoint
+   3. CrowdStrike Falcon  
+2. In the **Settings** > **Integrations** tab, configure your fetching integration instance.   
+   **Note:**  
+   Set **Fetch Incidents** to true and set the **Incident Type** to  `Malware Investigation and Response` (for Cortex XDR - you can use the classifier `Cortex XDR Incident Handler - Classifier` instead of the incident type).
+3. **Review the master playbook inputs**. These inputs control key features in the investigation process.
+
+
+
+#### Fetch Data from a SIEM
+If you prefer fetching incidents using a SIEM integration, you need to perform the following steps.  
+  
+1. **Before you start:** Set up your relevant EDR integration. For more details on supported EDR integrations, check the [Endpoint Integrations](#endpoint-integrations) section.
+2. Through your SIEM integration mapper, map the following fields to retrieve all required information:  
+   - Original alert ID mapped to  `External System ID` incident field.
+   - Incident type mapped to `External Category Name` incident field. (E.g. in CrowdStrike you have 2 different types - Detections and Incidents).
+   
+   If you do not want to update your current mapper,change the following playbook inputs to the prefered `Alert ID` and the `Incident Type` incident fields in the master playbook ([Malware Investigation and Response Incident Handler](https://xsoar.pan.dev/docs/reference/playbooks/malware-investigation--response-incident-handler)):
+     - `SIEMincidentFieldForID`  
+     - `SIEMincidentFieldForType` 
+3. Validate that relevant EDR integration is enabled.
+4. Ensure that relevant incoming incidents are assigned with 'Malware Investigation And Response' (through the SIEM Classifier).
+
+### Additional Integrations
+#### Mitre ATT&CK 
+The Mitre Att&ck integration enables easy classification for investigated incidents and better understanding of the current stage in the kill chain phase. The Mitre Att&ck integration ingests enriched information about the found techniques associated with the incident.  
+Information on findings are presented in the layout in the **Investigation Summary** and **Detailed Investigation** sections. Please **enable** this integration ( Optional - decide if you want to enable the fetching setting in advance).
+
+Related Playbook: [Mitre Attack - Extract Technique Information From ID](https://xsoar.pan.dev/docs/reference/playbooks/mitre-attack---extract-technique-information-from-id)
+
+#### [Palo Alto Networks WildFire - SandBox](https://xsoar.pan.dev/marketplace/details/Palo_Alto_Networks_WildFire) (Optional)
```

Comment:
@ssokolovich lets have one section for Sandbox with references to WildFire and FalconX

Reply:
Different from the additional Integration? Like we had with the Packs before I removed it? 
Or you prefer just highlighting that as a sandboxes section in the `Additional Integrations`